### PR TITLE
qm-rollback: add page

### PR DESCRIPTION
**Pull request by abhisheksrivastava177**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/qm-rollback.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# qm rollback
+
+> Rollback the VM state to a specified snapshot.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Rollback the state of the VM to a specified snapshot:
```
Comment (kbdharun) [inline pages/linux/qm-rollback.md:6]:
```suggestion
- Rollback the state of a specific VM to a specified snapshot:
```

File: pages/linux/qm-rollback.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+`qm rollback {{vmid}} {{snapname}}`
```
Comment (kbdharun) [inline pages/linux/qm-rollback.md:8]:
```suggestion
`qm rollback {{vm_id}} {{snap_name}}`
```
We use snake_case when using multiple words in command arguments

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

**Pull request merged**